src/components/App/App.js

```diff
@@ -12,18 +13,13 @@ function App() {
     setFavs([favObj, ...favs])
   }
 
-  // const removeFavorite = (id) => {
```

Comment:
It looks like you have this function operating in another component so you can remove this commented out code